### PR TITLE
Update the Devtools Node layout

### PR DESCRIPTION
**Pull request by jasonLaster**
Here's a replay with some thoughts. Theres at least one thing i will tidy up tomorrow

https://replay.io/view?id=0cdf1b09-d245-4be7-9c2e-e4ef45097bb9&point=14278816362184167063060290937553346&time=7196.719843290892&hasFrames=true

Screen Shot 2021-04-14 at 5 22 00 PM

### CODE REVIEW

File: src/ui/components/Header/Header.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ function HeaderTitle({ recordingId, editingTitle, setEditingTitle }) {
   }
 
   const { recordingTitle, title } = data.recordings?.[0] || {};
+  console.log(data, recordingTitle, title);
```
Comment (sergeichestakov) [inline src/ui/components/Header/Header.js:81]: did you mean to leave this console.log in here?
```suggestion
```

File: src/ui/components/SecondaryToolbox/index.js [hunk 2]
```diff
@@ -88,17 +88,19 @@ function InspectorPanel() {
   );
 }
 
-function SecondaryToolbox({ selectedPanel, setSelectedPanel, narrowMode }) {
+function SecondaryToolbox({ selectedPanel, setSelectedPanel, narrowMode, recordingTarget }) {
   const {
     userSettings: { show_react },
   } = hooks.useGetUserSettings();
+  const isNode = recordingTarget == "node";
```
Comment (sergeichestakov) [inline src/ui/components/SecondaryToolbox/index.js:95]: should this be using triple equals?
```suggestion
  const isNode = recordingTarget === "node";
```

File: src/ui/components/SecondaryToolbox/index.js [hunk 3]
```diff
@@ -88,17 +88,19 @@ function InspectorPanel() {
...
   return (
-    <div className="secondary-toolbox">
+    <div className={`secondary-toolbox ${isNode && "node"}`}>
```
Comment (sergeichestakov) [inline src/ui/components/SecondaryToolbox/index.js:97]: I typically use`classnames` when I need to conditionally applying class names and thoroughly enjoy it: https://www.npmjs.com/package/classnames

Just a suggestion!

**Review by sergeichestakov — APPROVED** [review]
Left a couple comments/suggestions but this looks great! 😄 

Only other thing I'd say here is that there seems to be a lot of special casing for Node. I saw you mentioned this in your replay as well. Don't have enough context on which languages you support and how their use cases differ but is there anything here that can be generalized? That way you can conditionally render based on a language "type" or specific use case rather than on the language itself because I imagine it can get pretty messy if each language has to render something different.

At Replit, for example, we have a config for each language with a bunch of fields, (e.g. `hasGit`, `hasInterpreter`, etc) each of which have implications for what gets shown in the IDE:

Screen Shot 2021-04-15 at 10 44 59 AM

You can see the full list here: http://eval.global.replit.com/languages

Curious to hear your thoughts! Not a blocker by any means for this PR, you can always follow up anyways

**Review by jaril — APPROVED** [review]
Apart from the items that @sergeichestakov brought up, looks good to me!

File: src/ui/components/Header/Header.js (hunk 1 above)
Comment (jasonLaster) [reply src/ui/components/Header/Header.js:81]: haha - sloppy :)

File: src/ui/components/SecondaryToolbox/index.js (hunk 3 above)
Comment (jasonLaster) [reply src/ui/components/SecondaryToolbox/index.js:97]: yep - we use classnames too, just moving quickly

File: src/ui/components/SecondaryToolbox/index.js (hunk 2 above)
Comment (jasonLaster) [reply src/ui/components/SecondaryToolbox/index.js:95]: hehe

**Comment by jasonLaster** [comment]
Thanks - I like the idea of having target attributes like `hasGraphics` as opposed to specific language checks. Another example of this will be libraries + frameworks such as React, Redux, React Profiling

**Pull request merged**